### PR TITLE
Fix WebSocket link with an explanation

### PR DESCRIPTION
**Pull request by pcheremu**
Current "Full example" was unworkable because we must match correct WebSocket with a Provider. Here a solution of this issue with description at the header.
https://linear.app/matterlabs/issue/QA-217/wrongunworkable-example-in-the-api-documentation

### CODE REVIEW

File: docs/dev/events.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 This is a quick tutorial on how to use zkSync events API. For the detailed description of events data types, please refer to [the documentation](../api/events.md).
 
-The feature is currently available only on the Ropsten testnet. The API is not yet fully stabilized and may change in the future.
+The feature is currently available only on the Ropsten and Rinkeby testnet. The API is not yet fully stabilized and may change in the future.
```
Comment (StanislavBreadless) [inline docs/dev/events.md]: > The feature is currently available only on the Ropsten and Rinkeby testnet

It is also available on mainnet (you provide the link to the mainnet below)

File: docs/dev/events.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+Make sure, you have selected correct WebSocket and DefaultProvider:
+```javascript
+F.e.:
+WebSocket("wss://ropsten-events.zkscan.io/") <-> getDefaultProvider("ropsten")
```
Comment (StanislavBreadless) [inline docs/dev/events.md]: Maybe better to show as a list of URLs? A string

`wss://ropsten-events.zkscan.io  -   ropsten  Provider` is not JS code.

File: docs/dev/events.md [hunk 3]
```diff
@@ -130,7 +139,7 @@ async function main() {
   // Get the provider. It's important to specify the correct network.
   const provider = await zksync.getDefaultProvider("ropsten");
   // Connect to the event server.
-  const ws = new WebSocket("wss://events.zksync.io/");
+  const ws = new WebSocket("wss://ropsten-events.zkscan.io/");
```
Comment (StanislavBreadless) [inline docs/dev/events.md]: I propose to change the provider to mainnet instead of changing the websocket url

Comment (pcheremu) [reply docs/dev/events.md]: @StanislavBreadless No problem, Just by my understanding, with highest probability customers will use any of test networks before they will switch them to use real funds.

File: docs/dev/events.md [hunk 4]
```diff
@@ -4,9 +4,9 @@ This is a quick tutorial on how to use zkSync events API. For the detailed descr
 
 The feature is currently available only on the mainnet, the Ropsten and Rinkeby testnet. The API is not yet fully stabilized and may change in the future.
 
-Make sure, you have selected correct WebSocket (Required) and Provider (Optional)*.
+Make sure, you have selected correct WebSocket (Required) for Network (Optional)*.
```
Comment (StanislavBreadless) [inline docs/dev/events.md]:
```suggestion
Make sure, you have selected the correct WebSocket URL for the network:
```

File: docs/dev/events.md [hunk 5]
```diff
@@ -4,9 +4,9 @@ This is a quick tutorial on how to use zkSync events API. For the detailed descr
...
 ```
-            WebSocket               Provider
+            WebSocket               Network
```
Comment (StanislavBreadless) [inline docs/dev/events.md:9]: Please organize this as a table in markdown

File: docs/dev/events.md [hunk 6]
```diff
@@ -2,7 +2,26 @@
...
+
+An example.
+```javascript
+...
+async function main() {
+  // Connect to the event server.
+  const ws = new WebSocket("wss://ropsten-events.zkscan.io/"); // Required
+...
+```
+*Provider is used just for user-friendly token formatting.
```
Comment (StanislavBreadless) [inline docs/dev/events.md]:
```suggestion
```

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**